benchmarks/README.md

```diff
@@ -0,0 +1,17 @@
+# Holoscan Benchmarks
+
+This directory contains applications, tools, and instructions to help measure
+the performance of pipelines built with Holoscan SDK.
+
+## Contents
+
+Any of the following may be covered under a benchmark project:
+- Holoscan SDK performance;
+- HoloHub application performance;
+- Tools for benchmarking Holoscan SDK applications not maintained in HoloHub.
```

Comment:
it says `not maintained in HoloHub`. do you mean it would be maintained outside of Holohub?

Reply:
Yes, here I mean to say that developers can use benchmarking tools to profile their own custom applications without necessarily adding those custom applications to HoloHub.

Do you have suggestions on how we can update the wording here to be more clear to new users?

Reply:
```suggestion
- Tools for benchmarking non-Holohub Holoscan SDK applications.
```